pkg/cvo/sync_worker.go

```diff
@@ -462,7 +467,7 @@ func (w *SyncWorker) Update(ctx context.Context, generation int64, desired confi
 		return w.status.DeepCopy()
 	}
 
-	work.Capabilities = capability.SetCapabilities(config, priorCaps)
+	work.Capabilities = capability.SetCapabilities(config, priorCaps, capability.GetCapabilitiesAsMap(w.alwaysEnableCapabilities))
```

Comment:
I think this belongs up where we check `if w.work != nil` in L457

Reply:
But I also wonder why you didn't add `alwaysEnableCapabilities` to the `Capabilities` in `w.work` instead of keeping it separate from the other capabilities maps?

Reply:
> I think you need to set a variable from w.alwaysEnableCapabilities up where we check if w.work != nil in L457

[I wanted](https://github.com/openshift/cluster-version-operator/pull/946#discussion_r1512788624) to consolidate the process of determining the list of implicitly enabled capabilities within one function: `SetCapabilities`. This way, the logic of compiling the list of  the prior capabilities should occur before the `SetCapabilities` call.

Reply:
Yeah, I was mixing up `w.work` with `work`.